### DISCUSSION TITLE
Simulate pages in the editor

### DISCUSSION THREAD

**Discussion by LucasFrra** [GrapesJS/grapesjs#5516] [Q&A]
Hello, 
I'd like to make my editor simulate pages, i.e. create a second page if the component is too big for the first etc, a bit like Google Docs. I've tried many things, to no avail. 
Would you have any ideas or is there a plugin for this that I haven't found? 
Thanks in advance

**Comment by javadsamiee** [comment]
Did you try https://grapesjs.com/docs/api/pages.html ?

**Comment by LucasFrra** [comment]
@javadsamiee I've read the documentation from top to bottom, but I really can't see how to create pages dynamically like a word processor like Word. If you have any ideas, I'd love to hear them!
